presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -334,6 +338,36 @@ the ``org.apache.hadoop.conf.Configurable`` interface from the Hadoop Java API,
 will be passed in after the object instance is created and before it is asked to provision or retrieve any
 encryption keys.
 
+Table Statistics
+----------------
+
+The Hive connector automatically collects basic statistics
+(``numFiles', ``numRows``, ``rawDataSize``, ``totalSize``)
+on ``INSERT`` and ``CREATE TABLE AS`` operations.
+
+The Hive connector can also collect column level statistics:
+
+============= ================================================================================================================
+Column Type                                            Collectible Statistics
+============= ================================================================================================================
+``TINYINT``   ``NUMBER_OF_NULLS``, ``MIN``, ``MAX``, ``NUMBER_OF_DISTINCT_VALUES``
```

Comment:
Should we use words here? This is for humans, and AFAIK, these constants like `NUMBER_OF_NULLS` are an internal detail of Presto.

Also, let's put min/max last, rather than in the middle of the "number of" stats.
```
``TINYINT``   number of nulls, number of distinct values, min/max values
``BOOLEAN``   number of nulls, number of true/false values